test/wamp-client-fs.spec.ts

```diff
@@ -29,16 +28,30 @@ describe(`the wamp client filesystem implementation`, () => {
         timeout: 30
     };
 
-    beforeEach(() => server().then(clientAndServer => {
-        wampRouter = clientAndServer.router;
-        connection = clientAndServer.connection;
-    }));
+    function retry(callback: () => boolean, delay: number = 10, attempts: number = 50): Promise<void> {
```

Comment:
as there is no use case of a custom `delay` value, it should be a constant rather than an argument.
we don't need `attempts` limitations at all, just a `.timeout(1000)` on the result will be fine.
The entire function can be replaced by a line that uses `bluebird-retry`